### PR TITLE
Updating based on Dipp additions

### PR DESCRIPTION
**Pull request by 0xrohan10**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -9,6 +9,15 @@ services:
       - "dbdata:/var/lib/postgresql/data"
     ports:
       - "5432:5432"
+  redis:
```
Comment (Haegin) [inline docker-compose.yml]: If you're adding default redis, should you add that to the circle CI config?

File: .env.example [hunk 2]
```diff
@@ -1,4 +1,7 @@
 PORT=3333
-POSTGRES_USER=postgres
-POSTGRES_HOST=127.0.01
-POSTGRES_DB=hello_world
+PP_KEYS=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_HOST=localhost
+POSTGRES_DB=
+DB_CONNECTION_STRING=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRESS_HOST}:5432/${POSTGRES_DB}
```
Comment (Haegin) [inline .env.example:7]: Sadly with `dotenv` I don't think interpolation works. `direnv` is fine with it, but that needs setting up at the system level.

**Review by Haegin — CHANGES_REQUESTED** [review]
I'm pretty sure interpolation won't work in .env files.

File: .env.example (hunk 2 above)
Comment (0xrohan10) [reply .env.example:7]: This is exactly how we have it on `offers.api`

**Review by Haegin — APPROVED** [review]

**Pull request merged**